web/src/app/app.module.ts

```diff
@@ -11,13 +11,15 @@ import { ButtonRowComponent } from './components/button-row/button-row.component
 import { ButtonComponent } from './components/button/button.component';
 import { PageNameComponent } from './components/page-name/page-name.component';
 import { DateNsecPipe } from './pipes/date-nsec.pipe';
+import { TimelineComponent } from './components/timeline/timeline.component';
 
 @NgModule({
   declarations: [
     AppComponent,
     MyPushesComponent,
     AllPushesComponent,
     OnePushComponent,
+    TimelineComponent,
     ButtonRowComponent,
     ButtonComponent,
     PageNameComponent,
```

Comment:
Can we keep the list sorted?

Reply:
Should this be done as part of my PR though, since the rest of the list isn't sorted as well? I don't want to cause merge conflicts

Reply:
There will be conflicts in this file anyway, I think it's fine to do it in the same PR.

Reply:
Got it!